style.css

```diff
@@ -1503,3 +1507,435 @@
   font-size: 23px;
   margin-left: 18px;
 }
+
+@media only screen and (min-width: 768px) {
+  .toolbar {
+    width: 100vw;
+  }
+
+  .menu-bar {
+    display: flex;
+    width: fit-content;
+    right: 120px;
+  }
+
+  .menu-bar img {
+    display: none;
+  }
+
+  .nav-menu {
+    display: flex;
+    list-style-type: none;
+  }
+
+  .nav-menu li {
+    color: #fff;
+    margin-left: 80px;
+  }
```

Comment:
For a better user experience on the website, kindly ensure that the content fits on the page without overflow.

Your design:
![image](https://user-images.githubusercontent.com/20554583/215264648-ff63bf47-4205-46d9-a56d-e60a776cab47.png)

Reply:
Done.